pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -76,6 +76,21 @@ type BundleSource struct {
 	// InLine is a simple string to append as the source data.
 	// +optional
 	InLine *string `json:"inLine,omitempty"`
+
+	// HTTPS downloads a PEM-Formatted trust bundle from an HTTPS server
+	// +optional
+	HTTPS *SourceHTTPS `json:"https,omitempty"`
+}
+
+// SourceHTTPS represents a target HTTPS URL that will serve a PEM-formatted trust bundle.
+type SourceHTTPS struct {
+	// URL is an HTTPS URL.
+	URL string `json:"url"`
+	// CABundle is the CA Bundle required to verify the target server's identity.
+	CABundle string `json:"caBundle"`
+	// PollInterval is the period at which the controller will check for changes on the server.
+	// +optional
+	PollInterval *string `json:"pollInterval,omitempty"`
```

Comment:
Should we instead change this to be `PollIntervalSeconds int64`? This is more inline with the Kubernetes field conventions.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units

We should also have a default here which is managed by OpenAPI schema (kubebuilder headers). https://book.kubebuilder.io/reference/markers/crd-validation.html